mysite/blog/models.py

```diff
@@ -54,6 +55,15 @@ def get_absolute_url(self):
             ],
         )
 
+    def get_top_four_similar_posts(self):
```

Comment:
This should be tested.

- Create a post with a tag.
- post.get_top_four_similar_posts() should return empty queryset.

- Create two posts with a tag.
- first_post.get_top_four_similar_posts() should have second post